src/components/dashboard/DashBody.js

```diff
@@ -58,7 +60,7 @@ const BuildWidgets = widgetArray =>
       'Clock': <Clock { ...widget } key={ widget.title } />
     }[ widget.type ]))
 
-class DashboardBody extends Component {
+class DashBody extends Component {
```

Comment:
So I know @jrob8577 tends to hate abbreviations...any reason as to why this was changed?